tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/Price/GET list of prices by CatalogueItemId.feature

```diff
@@ -0,0 +1,115 @@
+	When a GET request is made to retrieve the pricing for catalogue item with ID Sln3
+	Then a successful response is returned
+	And Prices are returned
+		| Type | ProvisioningType | CurrencyCode | Price  | PricingItemName | PricingItemDescription | PricingItemTierName | TimeUnitName | TimeUnitDescription |
+		| Flat | Declarative      | GBP          | 348.92 | bed             | per bed                | beds                | month        | per month           |
+		| Flat | OnDemand         | USD          | 567.32 | licence         | per license            | licenses            | year         | per year            |
+
+@7264
+Scenario: 4. Get a list of Tiered prices
+	When a GET request is made to retrieve the pricing for catalogue item with ID Sln4
+	Then a successful response is returned
+	And Prices are returned
+		| Type   | ProvisioningType | CurrencyCode | PricingItemName | PricingItemDescription | PricingItemTierName | TimeUnitName | TimeUnitDescription |
+		| Tiered | Patient          | EUR          | bed             | per bed                | beds                | year         | per year            |
+		| Tiered | Declarative      | AUZ          | consultation    | per consultation       | consultations       | year         | per year            |
+	And the Prices Tiers are returned
+		| Start | End | Price  | Section |
+		| 1     | 8   | 900.00 | 0       |
+		| 9     | 15  | 800.00 | 0       |
+		| 16    |     | 700.00 | 0       |
+		| 1     | 8   | 800.00 | 1       |
+		| 19    |     | 700.00 | 1       |
+
+@7264
+Scenario: 5. Get a list of flat and tiered prices
+	When a GET request is made to retrieve the pricing for catalogue item with ID Sln5
+	Then a successful response is returned
+	And Prices are returned
+		| Type   | ProvisioningType | CurrencyCode | Price  | PricingItemName | PricingItemDescription | PricingItemTierName | TimeUnitName | TimeUnitDescription |
+		| Flat   | OnDemand         | GBP          | 521.90 | licence         | per license            | licenses            | NULL         | NULL                |
+		| Tiered | Patient          | GBP          |        | sms             | per SMS                | SMS                 | year         | per year            |
+	And the Prices Tiers are returned
+		| Start | End | Price   | Section |
+		| 1     | 10  | 2100.93 | 0       |
+		| 11    |     | 1943.21 | 0       |
+
+@7264
+Scenario: 6. Get a single Flat Price for an additional service
+	When a GET request is made to retrieve the pricing for catalogue item with ID Sln1-001A001
+	Then a successful response is returned
+	And Prices are returned
+		| Type | ProvisioningType | CurrencyCode | Price  | PricingItemName | PricingItemDescription | PricingItemTierName | TimeUnitName | TimeUnitDescription |
+		| Flat | Patient          | GBP          | 199.99 | patient         | per patient            | patients             | month        | per month           |
+
+@7264
+Scenario: 7. Get a price with no solution
```

Comment:
If we go along with one of my previous comments regarding removing the `_verifier` I think this test could be changed to check for an empty list of prices.

Also the name of the scenario suggests solution but specifically it is a catalogue item.